queries/gotmpl/highlights.scm

```diff
@@ -0,0 +1,106 @@
+; Identifiers
+[
+  (field)
+  (field_identifier)
+] @property
+
+(variable) @variable
+
+; Function calls
+(function_call
+  function: (identifier) @function)
+
+(method_call
+  method:
+    (selector_expression
+      field: (field_identifier) @function))
+
+; Builtin functions
+(function_call
+  function: (identifier) @function.builtin
+  (#set! priority 101)
```

Comment:
Priority isn't necessary i'd say. Just move the queries down. The effect can also be seen with tree-sitter-cli 0.21.0

Reply:
I think that's meant to overwrite the highlighting of the YAML node this is injected to (which, as I wrote earlier, should be handled by setting the priority of the YAML node to `90`).